pkg/terraform/stages/platform/stages.go

```diff
@@ -63,6 +64,9 @@ func StagesForPlatform(platform string) []terraform.Stage {
 	case nonetypes.Name:
 		// terraform is not used when the platform is "none"
 		return []terraform.Stage{}
+	case external.Name:
```

Comment:
nit: maybe using the same file convention of `platformTypetypes.Name`:
```suggestion
	case externaltypes.Name:
```